### PR TITLE
Docs(loader): Improve bulk loader's description

### PR DESCRIPTION
**Pull request by bucanero**
Fix DOC-41

### CODE REVIEW

File: content/deploy/fast-data-loading/bulk-loader.md [hunk 1]
```diff
@@ -122,28 +138,21 @@ $ tree ./out
...
-
-# Read RDFs or JSON from the passed file.
-$ dgraph bulk -f <path-to-gzipped-RDF-or-JSON-file> ...
+{{% notice "note" %}}
+Each Dgraph Alpha must have a copy of the group's `p` directory output. 
+{{% /notice %}}
 
-# Read multiple RDFs or JSON from the passed path.
-$ dgraph bulk -f <./path-to-gzipped-RDF-or-JSON-files> ...
+![Bulk Loader diagram](/images/deploy/bulk-loader.png)
```
Comment (OmarAyo) [inline content/deploy/fast-data-loading/bulk-loader.md:149]: Two notes:

1. If you really want to add the diagram I suggest you add it after line 155
2. I suggest you reach out to our designer to convert the diagram to something better looking

File: content/deploy/fast-data-loading/bulk-loader.md [hunk 2]
```diff
@@ -122,28 +138,21 @@ $ tree ./out
...
+- another in the `./out/1` directory.
 
-```sh
-$ dgraph bulk --help # To see the available flags.
-
-# Read RDFs or JSON from the passed file.
-$ dgraph bulk -f <path-to-gzipped-RDF-or-JSON-file> ...
+{{% notice "note" %}}
+Each Dgraph Alpha must have a copy of the group's `p` directory output. 
+{{% /notice %}}
```
Comment (OmarAyo) [inline content/deploy/fast-data-loading/bulk-loader.md:147]: IMHO:

- we cite twice that we need to copy the `p` dir to each alpha server, thus we should remove line 151 and replace it with this notice bar.
- Move the notice bar after the diagram

**Review by OmarAyo — CHANGES_REQUESTED** [review]
after addressing the two comments LGTM

File: content/deploy/fast-data-loading/bulk-loader.md (hunk 1 above)
Comment (bucanero) [reply content/deploy/fast-data-loading/bulk-loader.md]: moved the diagram.
I'll check with the marketing team, but I still think that we can merge this PR with the current diagram, and update the image when available.

File: content/deploy/fast-data-loading/bulk-loader.md (hunk 2 above)
Comment (bucanero) [reply content/deploy/fast-data-loading/bulk-loader.md]: I'm keeping both mentions to reinforce the message that each server needs a copy of the `p` folder

**Review by aaroncarey — APPROVED** [review]

**Pull request merged**